ci/ci.py

```diff
@@ -343,18 +353,18 @@ def main(argv):
         make_release = True
         engine_artifacts = args.engine_artifacts or "archived"
     elif branch == "editor-dev":
-        engine_channel = "stable"
+        engine_channel = "editor-alpha"
```

Comment:
Not sure a bout this name?
What does this mean?
The editor-dev branch doesn't have an engine built?
So how does this map correctly to the stable engine sha1?